MDANSE/Tests/UnitTests/test_databases.py

```diff
@@ -202,18 +202,22 @@ def test_has_property(self):
         self.assertFalse(ATOMS_DATABASE.has_property("INVALID"))
 
     def test_info(self):
-        self.assertEqual(
-            "----------------------------------------------------------------------\n"
-            "                                  H                                   \n"
-            " property                                                         value\n"
-            "----------------------------------------------------------------------\n"
-            " electronegativity                                                  2.2\n"
-            " family                                                       non metal\n"
-            " nucleon                                                              0\n"
-            " symbol                                                               H\n"
-            "----------------------------------------------------------------------",
-            ATOMS_DATABASE.info("H"),
-        )
+        info_text = ATOMS_DATABASE.info("H")
+        properties = []
+        for nline, line in enumerate(info_text.split('\n')):
+            if nline == 1:
+                self.assertEqual(line.strip(), "H")
+            elif nline ==2:
+                tokens = line.split()
+                for keyword in {"property", "value", "unit"}:
+                    self.assertIn(keyword, tokens)
+            elif nline > 3:
+                tokens = line.split()
+                if len(tokens) > 2:
+                    properties.append(tokens[0])
+        for keyword in self.properties:
+            self.assertIn(keyword, properties)
```

Comment:
Possibly with lines broken up a bit better.
```suggestion
        info_text = ATOMS_DATABASE.info("H")
        lines = iter(info_text.splitlines())
        self.assertEqual(next(lines).strip(), "H")
        self.assertTrue({"property", "value", "unit"}.issubset(next(lines).split()))
        
        properties = {
            tokens[0] for line in lines if len(tokens := line.split()) > 2
        }
        self.assertTrue(set(self.properties) < properties)
```